README.md

```diff
@@ -514,6 +514,9 @@ IEEE/RSJ Int. Conf. Intelligent Robots and Systems (IROS), 2022. [YouTube](https
 - <a name="zhu2022learning"></a>Zhu, Z., Hou, J., Lyu, X.,  
 *[Learning Graph-embedded Key-event Back-tracing for Object Tracking in Event Clouds](https://openreview.net/pdf?id=hTxYJAKY85)*,  
 Thirty-sixth Conference on Neural Information Processing Systems (NeurIPS), 2022. [PDF](https://openreview.net/pdf?id=hTxYJAKY85), [code](https://github.com/ZHU-Zhiyu/Event-tracking).
+- <a name="Elshair22opteng"></a>El Shair, Z., Rawashdeh, S.A.,  
+*[High-temporal-resolution event-based vehicle detection and tracking](https://doi.org/10.1117/1.OE.62.3.031209)*,  
+Optical Engineering, 2022. [PDF](https://www.spiedigitallibrary.org/journals/optical-engineering/volume-62/issue-3/031209/High-temporal-resolution-event-based-vehicle-detection-and-tracking/10.1117/1.OE.62.3.031209.full), [dataset](http://sar-lab.net/event-based-vehicle-detection-and-tracking-dataset/).
```

Comment:
This PDF link is not needed, since the official link https://doi.org/10.1117/1.OE.62.3.031209 is provided and the publication is open access. Both links point to the same page, so the 2nd link is not needed.

Reply:
Thank you for pointing this out, Dr. Gallego. I have just addressed this issue in the latest commit.